pennylane/templates/subroutines/hilbert_schmidt.py

```diff
@@ -129,31 +133,30 @@ def __init__(self, *params, v_function, v_wires, u_tape, do_queue=True, id=None)
 
     @property
     def num_params(self):
-        return 1
+        return self._num_params
 
     @staticmethod
     def compute_decomposition(
         params, wires, u_tape, v_tape, v_function=None, v_wires=None
     ):  # pylint: disable=arguments-differ,unused-argument
-        r"""Representation of the operator as a product of other operators (static method)."""
+        r"""Representation of the operator as a product of other operators."""
```

Comment:
The arguments and the returns section seem to be missing here (some other templates have them).